.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
```

Comment:
From the above, other useful generic ones (full list [here](https://github.com/pre-commit/pre-commit-hooks)):
* `check-case-conflict`
* `check-executables-have-shebangs`
* `check-merge-conflict`
* `mixed-line-ending` with `args: [--fix=lf]` for Windows users